inline_scan.sh

```diff
@@ -347,11 +329,26 @@ start_analysis() {
 	    exit 1
 	fi
 
-    FULLTAG="${SCAN_IMAGES[0]}"
-    check_status_with_digest
+	get_scan_result_by_sha
+}
+
+get_repo_digest_id() {
+    # Check to see if repo digest exists
+    if [[ "${SYSDIG_IMAGE_ID}" == '123456890abcdefg' ]]; then
+        DIGESTS=$(docker inspect --format="{{.RepoDigests}}" "${SCAN_IMAGES[0]}")
+    fi
+
+    BASE_IMAGE=$(echo ${IMAGE_NAMES[0]} | cut -d / -f 2 | cut -d : -f 1)
+
+    for digest in "${DIGESTS[@]}"; do
+        if [[ ${digest} == *"${BASE_IMAGE}"* ]]; then
+            DIGEST=$(echo ${digest//]} | cut -d : -f 2)
+            SYSDIG_IMAGE_ID=${DIGEST}
+        fi
+    done
```

Comment:
I think this piece is not doing what it's supposed to do, and specially doesn't work with multiple values in the array, when docker inspect returns: 
```
[alpine@sha256:c19173c5ada610a5989151111163d28a67368362762534d8a8121ce95cf2bd5a alpine@sha256:e4355b66995c96b4b468159fc5c7e3540fcef961189ca13fee877798649f531a]
```

Since the result of docker inspect is going to be always surrounded by brackets I don't think bash is going to be able to iterate over it.